Source/Blazorise/Modules/JSUtilitiesModule.cs

```diff
@@ -53,7 +53,7 @@ public virtual ValueTask<bool> ParentHasClass( ElementReference elementRef, stri
 
     /// <inheritdoc/>
     public virtual ValueTask Focus( ElementReference elementRef, string elementId, bool scrollToElement )
-        => InvokeSafeVoidAsync( "focus", elementRef, elementId, scrollToElement );
+        => InvokeSafeVoidAsync( "focus", elementRef.Context is null ? null : elementRef, elementId, scrollToElement );
```

Comment:
What is the purpose of this null check? Can you explain?

Reply:
Because `ElementRef` is a struct. You need to explicitly send a null for `getRequiredElement` to use `elementId` instead.

![image](https://github.com/Megabit/Blazorise/assets/22283161/559789a7-8398-42b8-bbfb-6d51017f2306)